### PR TITLE
ampir stub, nf-test, snap

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: modules/nf-core/ampir/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    name "Test Process AMPIR"
+    script "../main.nf"
+    process "AMPIR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "ampir"
+
+    test("sarscov2 - bam") {
```
Comment (CarsonJM) [inline modules/nf-core/ampir/tests/main.nf.test:11]: It looks like the file used is actually candidatus_portiera_aleyrodidarum - proteome_fasta

File: modules/nf-core/ampir/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - bam - stub") {
```
Comment (CarsonJM) [inline modules/nf-core/ampir/tests/main.nf.test:36]: Change from sarscov2, similar to previous comment

**Review by CarsonJM — APPROVED** [review]

File: modules/nf-core/ampir/tests/main.nf.test (hunk 1 above)
Comment (sateeshperi) [reply modules/nf-core/ampir/tests/main.nf.test:11]: ah yes..thanks for spotting

**Comment by CarsonJM** [comment]
I just realized this was your PR @sateeshperi ! I am honored to have reviewed your PR after all the PRs you have reviewed for me

**Pull request merged**